certified-connectors/Tesseron ASM Basic Data/apiProperties.json

```diff
@@ -0,0 +1,42 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "instanceURL": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Instance URL",
+          "description": "Your Tesseron ASM instance URL",
+          "tooltip": "Provide the Tesseron ASM instance URL (without www.)",
+          "constraints": {
+            "required": "true"
+          }
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "apikey",
+          "description": "Your Tesseron ASM API key (Service: Ticket)",
+          "tooltip": "SProvide your Tesseron ASM API key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#218472",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('instanceURL')"
+        },
+        "templateId": "dynamichosturl",
+        "title": "instanceURL"
+      }
+    ],
+    "publisher": "Luithle + Luithle GmbH"
```

Comment:
@developmentLL stackOwner property is missing, please refer to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).